src/export/ExportFFmpeg.cpp

```diff
@@ -543,55 +610,67 @@ bool ExportFFmpeg::InitCodecs(AudacityProject *project)
          XO(
 /* i18n-hint: "codec" is short for a "coder-decoder" algorithm */
 "FFmpeg cannot find audio codec 0x%x.\nSupport for this codec is probably not compiled in.")
-            .Format( (unsigned int) mEncAudioCodecCtx->codec_id ),
+            .Format( (unsigned int) mEncAudioCodecCtx->GetCodecId() ),
```

Comment:
Since codec_id in the structure was not reassigned, should this be the local codecID instead?

Reply:
(Or translate the AUDACITY_ value to library value?)

Reply:
We can for sure